acme-dns-01-sandcats/index.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const https = require("https");
+const querystring = require("querystring");
+const dns = require("dns");
```

Comment:
Note that this file is part of an independent npm module, not under the Meteor build. I wasn't sure how to enable ES6 module syntax here and didn't want to spend time figuring it out, but if you can give me a hint I'm happy to do it.

(I didn't actually end up publishing this module to npm since it seems too specific to Sandstorm. I guess arguably I might as well fold it into the Meteor build rather than have it in this separate directory...)

Reply:
I sent a PR (#3302).